cmd/main.go

```diff
@@ -31,3 +39,21 @@ func echoBadRequestHandler(w http.ResponseWriter, r *http.Request) {
 		fmt.Fprintf(w, "query：%s\n", query)
 	}
 }
+
+func echoCheckIdHandler(w http.ResponseWriter, r *http.Request) {
+	validate := validator.New()
+
+	id := r.URL.Query().Get("id")
+	name := r.URL.Query().Get("name")
+	idInt, _ := strconv.Atoi(id)
```

Comment:
細かい要件がないので、これでも正解ですが
errは無視せずハンドリングしましょう。

errの場合も`0`になるので、後のバリデーションで弾かれるからおｋなのはわかるんですけどね。

Reply:
んで、ハンドリング処理で早期リターンとかするのがGoの風習だったりします。
こうすればvalidateのコストが浮きますしね。
```suggestion
	idInt, err := strconv.Atoi(id)
	if err != nil {
		log.Error("bad param.")
		http.Error(w, "Bad request", http.StatusBadRequest)
		return
	}
```

Reply:
ありがとうございます。
log.Errorはlogパッケージに無かったのですが、この場合はどれを使用すべきでしょうか？Panic、Fatal、Printのどれも適していないような気がして悩みました。

Reply:
あれ、Errorなかったかw
したらPrintかなぁ。

現場によってloggerはいろいろあるので、ログレベルをどうするか。を意識しておくとよさそうです。

Reply:
了解しました。プロジェクト入った時に意識するようにします。
内容修正しました！